### DISCUSSION TITLE
Reimagining the process for adding a country

### DISCUSSION THREAD

**Discussion by Abbe98** [govdirectory/website#409] [Ideas]
What if all of our country queries lived over at Wikidata and when the community makes changes and/or marks something as complete a bot picks up the change and makes a pull request based off the on-wiki change?

## Motivation

Currently I think that there is a disconnect between the work happening on Wiki and the implementation of that work happening here, but given the little metadata beyond queries needed to publish a country I don't think there is a reason why all of that couldn't live on wiki. Not only would we get notified of progress in a structured and actionable way but contributors would see a direct impact of their work and gain the ability to go much father in the publishing without touching Github. We could also have the bot make various quality checks, is the number of orgs correct? does the query pass? etc.

## Implementation

The current metadata and queries for countries would need to live on Wiki in a structured way, probably using templates, a bot would watch these country pages and upon a change it would make a new branch and pull request on Github, invalidate old pull requests, and finally notify the Wikidata contributor about the request.

I can't deny this would be a big chunk of work but it would be interesting to hear others thoughts on this approach.

**Comment by Ainali** [comment]
If we had a bot making some quality checks, then this sounds like an intriguing idea!

I have no idea how much work it would need or what is even needed, so I won't sign someone up for doing this work, but also not stop anyone with the inspiration.

**Reply by Abbe98** [reply]
Work wise I think it would need to be in a grant "work package" as some of the other things we imagine.
